### ISSUE TITLE
Has anyone installed & configured this successfully with examples?

### ISSUE THREAD

**Issue by gurudog** [magento/magento2-samples#81]
I would love to be able to try this module out on one of my Magento 2 sites. I want the 'add to cart' button to take the visitor to an external affiliate site for checkout. 

If you have done it or think you can set this up for me - hit me up.

**Comment by rparsi-commer** [comment]
@gurudog 
I think this repo has been abandoned, as the last commit is almost a full year ago.
Honestly the examples are mostly useless as they don't demonstrate anything similar
to typical real world scenarios (ie how to use the EAV database structure, the "new"
EntityManager class, etc).

**Comment by thomas-blackbird** [comment]
Yeah Magento expects you to build beautiful code, but don't tell you how...

**Comment by rparsi-commer** [comment]
@thomas-blackbird @gurudog 
This is unrelated question but I can't find any answers on my own:
What are the exact (meaning linux shell commands) steps for building the UI "stuff",
for a plain/vanilla (no custom theme, using defaults for everything) magento 2.1 application?
